Libraries/Utilities/MatrixMath.js

```diff
@@ -464,10 +464,10 @@ var MatrixMath = {
 
       // Solve the equation by inverting perspectiveMatrix and multiplying
       // rightHandSide by the inverse.
-      var inversePerspectiveMatrix = MatrixMath.inverse3x3(
+      var inversePerspectiveMatrix = MatrixMath.inverse(
         perspectiveMatrix
       );
-      var transposedInversePerspectiveMatrix = MatrixMath.transpose4x4(
```

Comment:
inverse3x3 and transpose4x4 don't exist on MatrixMath object. Also, this code is only reachable when the platform is android, as can be seen in processTransform.js:89.